client/src/containers/section/page/networks/network.tsx

```diff
@@ -0,0 +1,109 @@
+'use client';
+import { useState } from 'react';
+
+import { ArrowRight, Calendar, FolderOpen, Globe2 } from 'lucide-react';
+
+import { cn } from '@/lib/classnames';
+
+import {
+  ProjectListResponseDataItem,
+  OrganizationListResponseDataItem,
+} from '@/types/generated/strapi.schemas';
+
+import { Switch } from '@/components/ui/switch';
+export default function Network({
+  id,
+  attributes,
+  type,
+}: ProjectListResponseDataItem & OrganizationListResponseDataItem & { type: string }) {
+  const [checked, setChecked] = useState(false);
+  if (!id) return null;
+  const { name, short_description: shortDescription } = attributes || {};
+
+  const openDetails = () => {
+    // TOOD: Open details
+  };
+
+  const Icons = () => {
+    if (type === 'project') {
+      const {
+        project_type,
+        start_date: startDate,
+        end_date: endDate,
+        region_of_interventions,
+      } = attributes || {};
+      const projectType = project_type?.data?.attributes?.name;
+      const regionName = region_of_interventions?.data?.map((r) => r.attributes?.name).join(', ');
+      return (
+        <div className="flex gap-4">
+          {projectType && (
+            <div className="flex gap-2">
+              <FolderOpen className="h-6 w-6" />
+              <div className="text-base text-slate-500">{projectType}</div>
+            </div>
+          )}
+          {startDate && (
+            <div className="flex gap-2">
+              <Calendar className="h-6 w-6" />
+              <div className="text-base text-slate-500">
+                {startDate}
+                {endDate ? ` / ${endDate}` : ''}
+              </div>
+            </div>
+          )}
+          {regionName && (
+            <div className="flex gap-2">
+              <Globe2 className="h-6 w-6" />
+              <div className="text-base text-slate-500">{regionName}</div>
```

Comment:
This could possibly be very long. I wonder if we couldn't ellipse the text and add a tooltip to show the full text.